test/standalone/Soil/soil_test_3d.jl

```diff
@@ -11,6 +11,7 @@ import ClimaLand
 import ClimaLand.Parameters as LP
 
 for FT in (Float32, Float64)
+    #=
```

Comment:
commented out these tests since they compare the output of exp_tendency! with an analytic expression, but exp_tendency! does not do anything if there is no lateral flow.